tools/cli/go-whisk-cli/commands/action.go

```diff
@@ -850,6 +851,39 @@ func printActionDeleted(entityName string) {
             }))
 }
 
+// Check if the specified action is a web-action
+func IsWebAction(client *whisk.Client, qname *QualifiedName) bool {
+    var isWebAction = false;
+
+    savedNs := client.Namespace
+    client.Namespace = qname.namespace
+    fullActionName := "/" + qname.namespace + "/" + qname.entityName
+
+    action, _, err := client.Actions.Get(qname.entityName)
+    if err != nil {
+        whisk.Debug(whisk.DbgError, "client.Actions.Get(%s) error: %s\n", fullActionName, err)
+        whisk.Debug(whisk.DbgError, "Unable to obtain action '%s' for validation\n", fullActionName)
+    } else {
+        weVal := getValue(action.Annotations, "web-export")
```

Comment:
IMHO the extra checking benefit is two-fold.  1) to handle any kind of web-export annotation value (string, int, boolean, etc), and 2) discrete logging of the web-export value processing so we'll know the exact reason why the target action failed the check (i.e. not a single try/catch that logs all issues equally).